### PR TITLE
Update centralized-deployment.md

### PR DESCRIPTION
**Pull request by mattgeim**
Updating SSO in centralized deployment, removing preview status.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[c9fbd0b](https://github.com/OfficeDev/office-js-docs-pr/commits/c9fbd0b7c8a1fb66d93dce42e3a8498c0e979377)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/publish/centralized-deployment.md](https://github.com/OfficeDev/office-js-docs-pr/blob/mattgeim-patch-1/docs/publish/centralized-deployment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/publish/centralized-deployment?branch=pr-en-us-2186) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C9%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202009170100216022-2186%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=at5ls%2BHs6pfQo1KWX8IVNIUFZ7slDEhgA0YTBUBrfA0%3D&st=2020-09-17T00%3A56%3A00Z&se=2020-10-18T01%3A01%3A00Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/publish/centralized-deployment.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ If your organization meets all requirements, complete the following steps to pub
     ![Manage who has access and deployment method in Microsoft 365 admin center](../images/manage-users-deployment-admin-center.png)
 
     > [!NOTE]
-    > A [single sign-on (SSO)](../develop/sso-in-office-add-ins.md) system for add-ins is currently in preview and should not be used for production add-ins. When an add-in using SSO is deployed, the users and groups assigned are also shared with add-ins that share the same Azure App ID. Any changes to user assignments are also applied to those add-ins. The related add-ins are shown on this page. For SSO add-ins only, this page displays the list of Microsoft Graph permissions that the add-in requires.
+    > Add-ins that utilize [single sign-on (SSO)](../develop/sso-in-office-add-ins.md) will prompt the admin to consent to the scopes listed in the add-in manifest.  If the same backing service is used across multiple add-ins(the same Azure App ID is used with SSO in different add-ins), the scopes for each add-in will be prompted for consent with each deployment. This page will aslo display the list of permissions that the add-in requires.
```
Comment (davidchesnut) [inline docs/publish/centralized-deployment.md:77]:
```suggestion
    > Add-ins that utilize [single sign-on (SSO)](../develop/sso-in-office-add-ins.md) will prompt the admin to consent to the scopes listed in the add-in manifest.  If the same backing service is used across multiple add-ins (the same Azure App ID is used with SSO in different add-ins), the scopes for each add-in will be prompted for consent with each deployment. This page will also display the list of permissions that the add-in requires.
```

**Review by davidchesnut — CHANGES_REQUESTED** [review]
Looks good, just a couple typos...

**Comment by o365devx** [comment]
Docs Build status updates of commit _[8829552](https://github.com/OfficeDev/office-js-docs-pr/commits/8829552f362a1c196674d17186daac6cf763d4d6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/publish/centralized-deployment.md](https://github.com/OfficeDev/office-js-docs-pr/blob/mattgeim-patch-1/docs/publish/centralized-deployment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/publish/centralized-deployment?branch=pr-en-us-2186) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C9%5C21%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202009211929401853-2186%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=KGO4JFHL%2FrYbcoeo%2B3XPq2Bv2LKb3ps5AagIQFpAdfA%3D&st=2020-09-21T19%3A25%3A14Z&se=2020-10-22T19%3A30%3A14Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by davidchesnut — APPROVED** [review]
Looks good!

**Pull request merged**